src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -78,6 +99,65 @@ const LessonDetails = () => {
     })
   }
 
+  const getCooperation = useCallback(() => {
+    return cooperationService.getCooperationById(id)
+  }, [id])
+
+  const { data: cooperation } = useQuery({
+    queryFn: getCooperation,
+    queryKey: ['cooperation', id]
+  })
+
+  useEffect(() => {
+    if (!cooperation?.sections) return
+
+    cooperation.sections?.forEach((section) => {
```

Comment:
`sections` field is defined right now becasue of that ↑ `if` statement

```suggestion
    cooperation.sections.forEach((section) => {
```

Reply:
fixed